src/index.ts

```diff
@@ -130,7 +130,7 @@ function compileGlob(glob: string) {
   if (!relativeImportRE.test(glob)) {
     glob = './' + glob
   }
-  if (!glob.endsWith('*') && !/\.[tj]sx?/.test(glob)) {
+  if (!glob.split('/').pop()?.includes('*')) {
```

Comment:
`.split` never produces an empty array, so we can use non-null assertion here

```suggestion
  if (!glob.split('/').pop()!.includes('*')) {
```